src/Portal/test/localEnv.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file is only for live test, which is not required to be checked in.

Reply:
Good catch